plist_utils.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"io/ioutil"
+	"log"
+	"reflect"
+
+	"howett.net/plist"
+)
+
+const (
+	bundleIDKey        = "CFBundleIdentifier"
+	versionNumberKey   = "CFBundleShortVersionString"
+	buildNumberKey     = "CFBundleVersion"
+	bundleIconFilesKey = "CFBundleIconFiles"
+)
+
+type plistInfo struct {
+	bundleID      string
+	versionNumber string
+	buildNumber   string
+	bundleIcons   []string
+}
+
+func (e *plistInfo) UnmarshalPlist(unmarshal func(interface{}) error) error {
+
+	var plistDict map[string]interface{}
+
+	if err := unmarshal(&plistDict); err != nil {
+		return err
+	}
+
+	bundleID := reflect.ValueOf(plistDict[bundleIDKey]).String()
```

Comment:
you should avoid using reflection if you can: https://www.reddit.com/r/golang/comments/57otqv/is_using_reflect_idiomatic_go/
we usually do something like this: https://github.com/bitrise-io/go-xcode/blob/master/plistutil/plistutil.go